components/br-stream/src/router.rs

```diff
@@ -492,6 +505,9 @@ impl StreamTaskInfo {
             min_resolved_ts: TimeStamp::max(),
             files: SlotMap::default(),
             flushing_files: SlotMap::default(),
```

Comment:
Maybe try [atomic](https://docs.rs/atomic/latest/atomic/struct.Atomic.html)?
(It seems that TiKV has already required this.)